tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -62,10 +105,19 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 	})
 
 	Context("When using dry-run option to create operator backed service", func() {
+
+		JustBeforeEach(func() {
+			preSetup()
+		})
+
+		JustAfterEach(func() {
+			cleanPreSetup()
+		})
+
 		It("should only output the definition of the CR that will be used to start service", func() {
 			// First let's grab the etcd operator's name from "odo catalog list services" output
 			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
-			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]`).FindString(operators)
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(operators)
```

Comment:
This modification is done because earlier when etcd operator was installed in a single namespace, output of `odo catalog list services` would be as below:
```sh
odo catalog list services
Operators available in the cluster
NAME                                CRDs
etcdoperator.v0.9.4     EtcdCluster, EtcdBackup, EtcdRestore
```
But since it's now being installed from clusterwide channel, it looks like:
```sh
odo catalog list services
Operators available in the cluster
NAME                                CRDs
etcdoperator.v0.9.4-clusterwide     EtcdCluster, EtcdBackup, EtcdRestore

```